constants/types.go

```diff
@@ -10,6 +10,11 @@ const (
 	ACLObject = "-rwxr-xr-x"
 )
 
+// Available acl name.
+const (
+	PublicBucketACL = "QS_ALL_USERS"
```

Comment:
Currently we don't need to consider ACL and policy things, keep them for later refactor.